firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -335,6 +332,11 @@ struct SMC {
         if (_pressure1 > PEDAL_THRESH || _pressure2 > PEDAL_THRESH ) 
         {
             DEBUG_PRINT("Brake Pedal Detected");
+            DEBUG_PRINT("Brake Pedal Detected");
```

Comment:
I think this is a duplicate, @ltbj.  We are replacing `Serial.print` statements in the repo .